mercure.rst

```diff
@@ -68,11 +68,17 @@ clients.
 An official and open source (AGPL) implementation of a Hub can be downloaded
 as a static binary from `Mercure.rocks`_.
 
-Run the following command to start it:
+On Linux and Mac, run the following command to start it:
 
 .. code-block:: terminal
 
-    $ ./mercure --jwt-key='!ChangeMe!' --addr='localhost:3000' --allow-anonymous --cors-allowed-origins='*'
+    SERVER_NAME=:3000 MERCURE_PUBLISHER_JWT_KEY="!ChangeMe!" MERCURE_SUBSCRIBER_JWT_KEY="!ChangeMe!" ./mercure run -config Caddyfile.dev
```

Comment:
-config or --config?

Same question for windows

Reply:
`-config` (it's a Caddy server option).